wicket-core/src/main/java/org/apache/wicket/settings/FrameworkSettings.java

```diff
@@ -68,7 +88,7 @@ public FrameworkSettings(final Application application)
 	 * Gets the Wicket version. The Wicket version is in the same format as the version element in
 	 * the pom.xml file (project descriptor). The version is generated by maven in the build/release
 	 * cycle and put in the /META-INF/MANIFEST.MF file located in the root folder of the Wicket jar.
-	 *
+	 * <p></p>
```

Comment:
Most probably you want the closing `</p>` to be at the end of the paragraph.

Reply:
Or. create an  empty paragraph to separate from next IJ complains here

Reply:
Done.